src/TimeService.Business/Commands/Stat/FindStatCommand.cs

```diff
@@ -245,7 +245,9 @@ public FindResultResponse<StatInfo> Execute(FindStatFilter filter)
 
       if (filter.DepartmentId.HasValue)
       {
-        userIds = FindDepartmentUsers(filter.DepartmentId.Value, filter.SkipCount, filter.TakeCount, out totalCount, errors);
+        DateTime filterByEntryDate = new DateTime(filter.Year, filter.Month, 1).AddMonths(1);
```

Comment:
why do you add month

Reply:
to get all users with createdAd date < filterByEntryDate